app/assets/stylesheets/application.scss

```diff
@@ -15,4 +15,3 @@
  */
 
 @import "materialize";
-@import "https://fonts.googleapis.com/icon?family=Material+Icons";
```

Comment:
this was causing an import issue in controller tests. and per https://github.com/mkhairi/materialize-sass#c-icons we can just have it in `application.html.erb` without needing it here. so i removed it